src/server.ts

```diff
@@ -41,7 +37,7 @@ async function bootstrap(samples: boolean) {
         // Bind DB connections to context
         app.context.influx = influx;
         app.context.pg = pg;
-        app.context.mqtt = mqtt;
+        //app.context.mqtt = mqtt;
```

Comment:
Why did you comment out the MQTT client here?

Reply:
Will be added back later